### DISCUSSION TITLE
Is initials a part of Autocomplete Attribute Tokens 1.3.5 WCAG?

### DISCUSSION THREAD

**Discussion by AccessibilityEnthusiastic** [w3c/wcag#3002] [Q&A] [answered]
Hi Everyone,

Do initials require autocomplete?

If yes, will it be under an additional-name or family-name, or nickname?

https://www.w3.org/TR/WCAG21/#input-purposes
![autocomplete](https://user-images.githubusercontent.com/115781693/217026387-b49d466a-d640-418e-8fc2-5be12e7923a7.png)

**Comment by patrickhlauke** [comment, marked as answer]
as for HTML 1.3.5 primarily leans on the HTML spec's `autocomplete` values https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill and there is no value for initials, the answer's "no"

**Reply by AccessibilityEnthusiastic** [reply]
@patrickhlauke What about for the Lastname field? Is it applicable?
![image](https://user-images.githubusercontent.com/115781693/218308658-65e2d7fe-bf3d-49dd-b73c-06920cf67286.png)

**Reply by patrickhlauke** [reply]
why wouldn't it be? https://www.w3.org/TR/WCAG21/#input-purposes
